src/modules/VirtConfig.rb

```diff
@@ -610,17 +570,6 @@ def LXCDialog
       Builtins.y2milestone("Start virtlogd.socket: %1", cmd)
       SCR.Execute(path(".target.bash"), cmd)
 
-      # Firewall stage - modify the firewall setting, add the xen bridge to FW_FORWARD_ALWAYS_INOUT_DEV
-      # Progress::NextStage();
-
-      # Configure firewall to allow xenbr+
```

Comment:
The code is commented (so not used) but also it is not supported at all by firewalld, so removed completely the setup.